internal/util/projutil/project_util.go

```diff
@@ -238,7 +267,17 @@ func IsOperatorAnsible() bool {
 	return err == nil || os.IsExist(err)
 }
 
+// IsOperatorHelm will return true when the type of the project is Helm
 func IsOperatorHelm() bool {
+	// If the project has the new layout we will check the type in the config file
+	if kbutil.HasProjectFile() {
+		cfg, err := kbutil.ReadConfig()
+		if err != nil {
+			log.Print(err, "Error reading config")
+		}
+		return PluginKeyToOperatorType(cfg.Layout) == OperatorTypeHelm
+	}
+	// todo(camilamacedo86): remove when the legacy layout is no longer supported
```

Comment:
Will the charts dir no longer be in the new layout?

Reply:
ditto above. We can scaffold a project without any api.